ops/testing.py

```diff
@@ -919,13 +924,14 @@ def get_workload_version(self) -> str:
         return self._backend._workload_version
 
     def set_model_info(self, name: Optional[str] = None, uuid: Optional[str] = None) -> None:
-        """Set the name and uuid of the Model that this is representing.
+        """Set the name and UUID of the model that this is representing.
 
-        This cannot be called once begin() has been called. But it lets you set the value that
-        will be returned by Model.name and Model.uuid.
+        This cannot be called once :meth:`begin` has been called. But it lets
+        you set the value that will be returned by :attr:`Model.name <ops.Model.name>`
+        and :attr:`Model.uuid <ops.Model.uuid>`.
 
-        This is a convenience method to invoke both Harness.set_model_name
-        and Harness.set_model_uuid at once.
+        This is a convenience method to invoke both :meth:`set_model_name`
+        and :meth:`set_model_uuid` at once.
```

Comment:
Nit that shouldn't block the merge, but applies in a few places and would be good to clean up over time...

I'd prefer to remove the "you" - and stick with the imperative style (similar to commit messages).

Something like:

```python
"""Set the name and UUID of the model this represents.

Cannot be called once :meth:`begin` has been called. Enables the value returned by `:attr:`Model.name <ops.Model.name>` and :attr:`Model.uuid <ops.Model.uuid>` to be controlled.

Convenience method to invoke both :meth:`set_model_name` and :meth:`set_model_uuid` at once.
"""

Another approach might be `s/you/the developer/g` but I still think the above is cleaner.

Reply:
Good call. I'm going to leave it for another PR so this one can avoid too much sprawl. Opened issue https://github.com/canonical/operator/issues/959 to track.